packages/rs-drive-proof-verifier/src/verify.rs

```diff
@@ -75,7 +75,7 @@ pub(crate) fn verify_tenderdash_proof(
 
     // Verify signature
     let sign_digest = commit
-        .sign_digest(
+        .calculate_sign_hash(
```

Comment:
we should continue to call it a sign_digest

Reply:
sign_digest is deprecated; deprecation message says to use calculate_sign_hash instead

Reply:
Unless you're saying we should rename the variable?

Reply:
What library?

Reply:
@lklimek #[deprecated = "replaced by calculate_sign_hash() to unify naming between core, platform and tenderdash"]

Reply:
I think we regressed here.

Reply:
@PastaPastaPasta yes, tenderdash-abci regressed.

Reply:
Reverting until we get a better method in tenderdash.

Reply:
I think we decided that we stick to core naming, so the "deprecated" message is correct, and `calculate_sign_hash` is correct name.